homeassistant/components/emulated_hue.py

```diff
@@ -0,0 +1,623 @@
+"""
+Support for local control of entities via an emulated Hue bridge.
+
+A sample configuration entry is given below:

```

Comment:
This entire block needs to be removed and put into docs instead, including the license. 

```python
"""
Support for local control of entities through Alexa.

For more details about this component, please refer to the documentation at
https://home-assistant.io/components/alexa_local_control/
"""
```